Home_Work/HW_2_project/HW_2_app/forms.py

```diff
@@ -0,0 +1,13 @@
+from django import forms
+
+from .models import Order, Product
+
+
+class ProductForm(forms.ModelForm):
+    class Meta:
+        model = Product
+        fields = '__all__'
```

Comment:
Можно было сделать так
```suggestion
        exclude =['add_date']
```